docs/application/native/guides/app-management/app-controls.md

```diff
@@ -560,9 +565,9 @@ To run a specific application control with some preconfigured parameters:
 
    > **Note**
    >
-   > Since Tizen 2.4, service applications are only allowed to be launched explicitly and the caller must be an application in the same package. Otherwise, the `app_control` instance returns an error.
+   > Since Tizen 2.4, service applications are allowed to launch explicitly and the caller must be an application in the same package. Otherwise, the `app_control` instance returns an error.
```

Comment:
@mijin85cho Based on the 'Note' at Line 63:
"
        > **Note**
	> Since Tizen 2.4, application controls used to launch service applications outside the current package are not supported. Because of this, the service application is only allowed to be launched explicitly by the application in the same package. All service applications are excluded from matches of implicit launch requests.
"
Its confirmed that applications will launch explicitly only, so writing 'only' is not mandatory here. But keeping 'only' in here is still fine as meaning for both will remain the same.